### DISCUSSION TITLE
Direct download of installation packages

### DISCUSSION THREAD

**Discussion by swt1me** [community/community#140038] [Mobile]
Direct download of installation packages for users who cannot use Google Play

**Comment by t1noo7** [comment]
Thinking the best way is to use the pre-extracted version. You can try by using another computer, install it and copy the installation folder to your computer.

**Comment by Ankitjarwall** [comment]
hello @OpenEdgeLines, 

1. Direct APK Installation:
If the APK is built and signed properly, users can download it directly from a hosted link (e.g., cloud storage or third-party websites). Make sure the users enable "Install unknown apps" for the source they are downloading from, whether it's their browser or file manager.

2.Handling APK Bundles (Split APKs):
For apps with multiple APK files (like split APKs), users can utilize tools such as APKMirror Installer or SAI (Split APKs Installer). These tools allow users to install apps with multiple APK files seamlessly, ensuring all bundles are installed together.

3.Alternative App Stores:
Apart from Google Play, users can install apps and update them from alternative app stores such as:
             * Samsung Galaxy Store
            * Mi App Store (for Xiaomi devices)
           * Aptoide
       These stores also offer updates for apps, which can be useful if Google Play isn't available for some users.
       
i would suggest to go with 3rd option, easy and simple.

hope you got your answer.
Thank you.
